src/v2/Components/NavBar/NavItem.tsx

```diff
@@ -11,12 +11,13 @@ import { AnalyticsSchema } from "v2/Artsy"
 import { useTracking } from "v2/Artsy/Analytics/useTracking"
 import { isFunction, isString } from "lodash"
 import React, { useEffect, useRef, useState } from "react"
-import { animated, config, useSpring } from "react-spring"
-import styled, { css } from "styled-components"
+import styled from "styled-components"
 import { position } from "styled-system"
 import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { MenuAnchor, NavItemPanel } from "v2/Components/NavBar/NavItemPanel"
 
 const Container = styled(Flex)`
+  position: relative;
```

Comment:
So the crux of this issue is that we aren't using a positioning engine for these popdowns. So we have to rely on CSS. The problem is that since we're using CSS and want to absolutely position these outside of their parents, we previously were turning on and off relative. We can't do that anymore because the `UnfocusableAnchor` is being used to cover up the `HitArea`.